Makefile

```diff
@@ -32,16 +36,16 @@ OBJS =	idris-commands.elc		\
 .el.elc:
 	$(BYTECOMP) $<
 
-build: $(OBJS)
+build: getdeps $(OBJS)
 
-test:
+test: getdeps
 	$(BATCHEMACS) -L . -l ert -l idris-tests.el -f ert-run-tests-batch-and-exit
 
 clean:
 	-rm -f $(OBJS)
 	-rm -f test-data/*ibc
 
 getdeps:
-	$(BATCHEMACS) -eval '(progn (package-refresh-contents) (package-install (quote prop-menu)))'
+	$(BATCHEMACS) -eval '(progn (package-refresh-contents) (unless (package-installed-p (quote prop-menu)) (package-install (quote prop-menu)) ))'
```

Comment:
The space at the end of the line between closing parens is not good Lisp style. Ending parens should be in one gigantic pile :-)))))))